english/forest/fire/fight/forest/forest.md

```diff
@@ -0,0 +1,2 @@
+To take your spear, wash of the blood of the bear and go back into the forest. After a while where you always missed the quite shy deer that is in sight occasionally you think it might be better to climb on a tree and 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.